### DISCUSSION TITLE
During the feature extraction process, how could I set the param from a subset of another columns.

### DISCUSSION THREAD

**Discussion by lk1983823** [blue-yonder/tsfresh#1038] [Q&A]
For example, I want to use "number_crossing_m" to a column named "x1". So I have to set a parameter "m" to it. However, for the param "m", I want to let it be the mean of another column like "x2" or the "x1" itself. How should I do it ? 

Thanks.

**Comment by sophie-234** [comment]
Also posted in #1040.
